lib/src/main/java/com/thenewboston/data/dto/bankapi/invalidblockdto/request/PostInvalidBlockRequest.kt

```diff
@@ -0,0 +1,63 @@
+package com.thenewboston.data.dto.bankapi.invalidblockdto.request
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class PostInvalidBlockRequest(
+
+    @SerialName("message")
+    val message: PostInvalidBlockMessage,
+
+    @SerialName("node_identifier")
+    val nodeIdentifier: String,
+
+    @SerialName("signature")
+    val signature: String,
+)
+
+@Serializable
+data class PostInvalidBlockMessage(
+
+    @SerialName("block")
+    val block: PostInvalidBlockMessageBlock,
+
+    @SerialName("block_identifier")
+    val blockIdentifier: String,
+
+    @SerialName("primary_validator_node_identifier")
+    val primaryValidatorNodeIdentifier: String,
+)
+
+@Serializable
+data class PostInvalidBlockMessageBlock(
+
+    @SerialName("account_number")
+    val accountNumber: String,
+
+    @SerialName("message")
+    val message: PostInvalidBlockMessageBlockMessage,
+
+    @SerialName("signature")
+    val signature: String,
+)
+
+@Serializable
+data class PostInvalidBlockMessageBlockMessage(
```

Comment:
If you can think of a better name, let me know :smile:

We should think of a good way to handle all the different DTOs anyway, and maybe a consistent naming scheme.

Reply:
See https://thenewboston.com/bank-api/invalid-blocks

Reply:
Name it as `block` This is what the API calls it.

Reply:
The problem is that the API has all kinds of different structures named `message` and `block` so I don't want to use those generic names for this specific data structure. What do you think?

We should find a good way to handle all those different DTOs. That's why I created #86 as a first step.

Reply:
I see the problem now. Yeah it seems like we could solve this issue with using sealed classes for common data structures like `block` or `bank`